<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KLineOverviewWidget</class>
 <widget class="QWidget" name="KLineOverviewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_center">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_date">
          <property name="text">
           <string>2025-12-03</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_hours">
          <property name="text">
           <string>星期/小时</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_head_value">
          <property name="text">
           <string>数值</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_value">
          <property name="text">
           <string>123.123</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_head_open">
          <property name="text">
           <string>开盘</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_open">
          <property name="text">
           <string>123.123</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_head_close">
          <property name="text">
           <string>收盘</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_close">
          <property name="text">
           <string>123.123</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_head_high">
          <property name="text">
           <string>最高</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_high">
          <property name="text">
           <string>123.123</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_head_low">
          <property name="text">
           <string>最低</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_low">
          <property name="text">
           <string>123.123</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_head_change_percent">
          <property name="text">
           <string>涨幅</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_change_percent">
          <property name="text">
           <string>+3.2%</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_head_amplitude">
          <property name="text">
           <string>振幅</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_amplitude">
          <property name="text">
           <string>+6.2%</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_head_volume">
          <property name="text">
           <string>成交量</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_volume">
          <property name="text">
           <string>123.123</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_head_amount">
          <property name="text">
           <string>成交额</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_amount">
          <property name="text">
           <string>123.123</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_head_turnover_rate">
          <property name="text">
           <string>换手率</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_turnover_rate">
          <property name="text">
           <string>3.3%</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_head_volume_ratio">
          <property name="text">
           <string>量比</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_volume_ratio">
          <property name="text">
           <string>1.0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="label_head_kline_status">
          <property name="text">
           <string>k线状态</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_kline_status">
          <property name="text">
           <string>涨停/跌停</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
